doc/building.md

```diff
@@ -39,18 +39,26 @@ technical expertise, a fair number of dependencies on external software, and
 reasonably powerful hardware.
 
 If you just want to use the JDK and not build it yourself, this document is not
-for you. See for instance [OpenJDK installation](
-http://openjdk.org/install) for some methods of installing a prebuilt
-JDK.
+for you. See for instance [OpenJDK installation](http://openjdk.org/install)
```

Comment:
Switch to https? Same for other uses of http:// later in the file?

Reply:
There is a separate PR posted for that specific change already.